modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,81 @@
+[id="cluster-logging-release-notes-5-1-0"]
+= OpenShift Logging 5.1.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0].
+
+[id="openshift-logging-5-1-0-new-features-and-enhancements"]
+== New features and enhancements
+
+OpenShift Logging 5.1 now supports {product-title} 4.7 and later running on:
+
+* IBM Power Systems
+* IBM Z and LinuxONE
+
+This release adds improvements related to the following components and concepts.
+
+* As a cluster administrator, you can use Kubernetes pod labels to gather log data from an application and send it to a specific log store. You can gather log data by configuring the `inputs[].application.selector.matchLabels` element in the `ClusterLogForwarder` custom resource (CR) YAML file. You can also filter the gathered log data by namespace.
+(link:https://issues.redhat.com/browse/LOG-883[*LOG-883*])
+
+* This release adds the following new `ElasticsearchNodeDiskWatermarkReached` warnings to the OpenShift Elasticsearch Operator (EO):
+ - Elasticsearch Node Disk Low Watermark Reached
+ - Elasticsearch Node Disk High Watermark Reached
+ - Elasticsearch Node Disk Flood Watermark Reached
+
++
+The alert applies the past several warnings when it predicts that an Elasticsearch node will reach the `Disk Low Watermark`, `Disk High Watermark`, or `Disk Flood Stage Watermark` thresholds in the next 6 hours. This warning period gives you time to respond before the node reaches the disk watermark thresholds. The warning messages also provide links to the troubleshooting steps, which you can follow to help mitigate the issue. The EO applies the past several hours of disk space data to a linear model to generate these warnings.
+(link:https://issues.redhat.com/browse/LOG-1100[*LOG-1100*])
+
+* JSON logs can now be forwarded as JSON objects, rather than quoted strings, to either Red Hat's managed Elasticsearch cluster or any of the other supported third-party systems. Additionally, you can now query individual fields from a JSON log message inside Kibana which increases the discoverability of specific logs. For more information, see the Exported fields topic in the OpenShift Logging documentation.
```

Comment:
@sichvoge @alanconway Please review and approve these release notes today.

Reply:
@rolfedh why do we point to the exported fields topic instead if the section that explains how to enable and use our new JSON feature?

Reply:
/lgtm

Reply:
@sichvoge @alanconway @anpingli @rolfedh We cannot add hard-link inside the RN module. So, removed the entire "For more information .." sentence, as we cannot give link to the section that explains how to enable and use our new JSON feature. PTAL and kindly provide LGTM. Thanks.

Reply:
> /lgtm

Thanks @alanconway

Reply:
@sichvoge @alanconway @anpingli @rolfedh We cannot add hard-link inside the RN module. So, removed the entire "For more information .." sentence, as we cannot give link to the section that explains how to enable and use our new JSON feature. PTAL and kindly provide LGTM. Thanks.